source/administration/object-management.rst

```diff
@@ -220,6 +220,33 @@ Conversion to or from XML may be required for importing rules created on S3 or s
 
 See :ref:`minio-lifecycle-management` for more complete documentation.
 
+Target Bucket Configuration for Tiering
+---------------------------------------
+
+Keep in mind the following considerations when configuring the target bucket for tiering:
+
+- The target bucket *can* have its own set of object management rules different from the source.
+  For example, the source bucket may have object locking defined while the target bucket does not.
+- While enabling tiering requires :ref:`versioning <minio-bucket-versioning>` on the source bucket, the target bucket does *not* require versioning.
+  Enabling versioning on both the source and target buckets may lead to unexpected results.
```

Comment:
Is there anything useful to say about _what sort_ of unexpected results?

Reply:
Mostly that we already UUID on the remote, so versioning makes no sense.

instead when you delete a tiered object on the source, you get a deletemarker on the remote, so your storage efficiency drops.

At worst you end up with delete markers in both places. ILM eventually cleans up the old metadata on the source, but the remote is still versioned and w/o ILM it just sticks around.

The worst case is ILM on the remote that is not on the source, and then that tears apart your data.

Reply:
Added Ravind's comments to a completely reworked section.